src/free-trial/tax/woocommerce-tax/setup.tsx

```diff
@@ -101,25 +102,14 @@ export const Setup: React.FC< SetupProps > = ( {
 			),
 			content: <StoreLocation { ...stepProps } />,
 		},
-		{
-			key: 'plugins',
-			label: pluginsToActivate.includes( 'woocommerce-services' )
-				? __( 'Install Jetpack and WooCommerce Tax', 'woocommerce' )
-				: __( 'Install Jetpack', 'woocommerce' ),
-			description: __(
-				'Jetpack and WooCommerce Tax allow you to automate sales tax calculations',
-				'woocommerce'
-			),
-			content: <Plugins { ...stepProps } />,
-		},
 		{
 			key: 'connect',
-			label: __( 'Connect your store', 'woocommerce' ),
+			label: __( 'Automate Taxes', 'woocommerce' ),
```

Comment:
AFAIK, we don't handle bridge i18n properly right now but I think we should use `wc-calypso-bridge` text domain here. What do you think?